plugins/communications-app/SubjectForm/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Form } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { useSelector, useDispatch } from '@communications-app/src/components/bulk-email-tool/bulk-email-form/BuildEmailFormExtensible/context';
+import { actionCreators as formActions } from '@communications-app/src/components/bulk-email-tool/bulk-email-form/BuildEmailFormExtensible/context/reducer';
+
+import messages from './messages';
+
+const SubjectForm = () => {
+  const intl = useIntl();
+  const formData = useSelector((state) => state.form);
+  const dispatch = useDispatch();
+  const { subject, isFormSubmitted } = formData;
+
+  const handleChangeEmailSubject = ({ target: { value } }) => {
+    dispatch(formActions.updateForm({ subject: value }));
+  };
+
+  const subjectValidation = subject.length > 0;
```

Comment:
Same comment about boleans:


```suggestion
  const isSubjectValid = subject.length > 0;
```